Advice

CarolynBuck-Gengler opened the pull request:
I didn't realize until after editing .gitignore and just rotely git adding and git committing it that of course it would show up here. (I had had a misunderstanding that had resulted in the creation of a file that I wanted to ignore when I did git status; in retrospect I should have just continued to ignore it myself instead of editing .gitignore... misunderstanding resolved, FWIW.) 

Anyway, if there's a way on your end to just ignore that edit and commit, that would be great!

redrambles commented:
Hey @CarolynBuck-Gengler !

This is great! The only thing is that an additional line was added to the .gitignore file - `last31commands` which we don't need. ([You can see it here](https://github.com/skillcrush/105-practice/pull/1237/files))

Once you have removed that line - please add and commit and then push up your advice branch once more, which will automatically update this pull request. Then we should be all set to merge in your code!

Let us know if you have any questions.

Cheers,
Ann :)

CarolynBuck-Gengler commented:
Thanks @redrambles!

I have deleted that line from .gitignore and added and committed the revised .gitignore on my advice branch.

I assume before I push my advice branch I should do the git fetch upstream again? I know that there has been some merging of other students' pull requests since I last did mine. And if so, do I do that from my advice branch or my master branch? some of these little details are getting tangled up.

Thanks,
Carolyn

acodeinprogress commented:
Hey @CarolynBuck-Gengler,

Great job! Your code is all [merged in](https://media.tenor.com/images/4a6158ad4f30c71f4fecf224cec12eb1/tenor.gif).


xoxo,
Sharon
